### PR TITLE
Fixes #9663: ncf Editor: Highlight condition box if a condition is active

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/9663

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
It will always be open here, don't we want to open it if there is a condition set ?

**Comment by RaphaelGauthier** [comment]
Commit modified

**Comment by RaphaelGauthier** [comment]
Commit modified

File: builder/js/ncf.js [hunk 1]
```diff
@@ -188,6 +188,9 @@ app.controller('ncf-builder', function ($scope, $modal, $http, $log, $location,
   $scope.selectedMethod;
   // Are we authenticated on the interface
   $scope.authenticated = false;
+  // Open/Close by default the Conditions box
+  $scope.conditionIsOpen;
```
Comment (VinceMacBuche) [inline builder/js/ncf.js]: You should define "false" as default value, to prevent undefined

**Comment by RaphaelGauthier** [comment]
Commit modified

**Pull request merged**